cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -74,6 +75,7 @@ type CMServer struct {
 	NamespaceSyncPeriod               time.Duration
 	PVClaimBinderSyncPeriod           time.Duration
 	VolumeConfigFlags                 VolumeConfigFlags
+	TerminalPodGCThreshold            int
```

Comment:
s/Terminal/Terminated/
(everywhere in this PR including doc)
(terminal means something is going to die, whereas terminated means it has already died)